### DISCUSSION TITLE
Plan to support the migration of issues to discussions?

### DISCUSSION THREAD

**Discussion by pattacini** [community/community#23181] [New to GitHub]
<p>I’m very curious and really looking forward to getting my hands on GitHub Discussions announced during <a href="https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/" rel="nofollow noopener">GitHub Satellite 2020, </a>which by the way are now in beta and available only to a few of our communities.</p>
<p>I’ve got a question about this new component though. As we’ve been using the <a href="https://github.com/robotology/QA" rel="nofollow noopener">issue system of a specific repository for our Q&amp;A purposes</a>, I’m wondering if GitHub has already planned to support the migration of issues to discussions. This way, we could easily merge our current database of Q&amp;A in the (soonish) newly available service.</p>

**Comment by pattacini** [comment]
<p>According to this <a href="https://www.infoq.com/news/2020/05/GitHub-codespaces-discussions" rel="nofollow noopener">online resource</a>, we will have that:</p>
<blockquote>
<p>Confirming the strict relationship between Issues and Discussions, GitHub enables to convert back and forth between the two formats.</p>
</blockquote>

**Comment by pattacini** [comment]
<p>Somewhat related topic where it seems there’s a way to perform the conversion.<br>
<aside class="quote quote-modified" data-post="3" data-topic="131625">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/what-access-permissions-are-required-to-move-issues-to-discussions-and-vice-versa/131625/3">What access permissions are required to move issues to discussions and vice versa?</a> <a class="badge-wrapper  bullet" href="/c/github-help/how-to-use-git-and-github/21"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #ffd1ac;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Ask and answer questions regarding Git and GitHub.">How to use Git and GitHub</span></a>
  </div>
  <blockquote>
    You can move issues to discussions, and triage is all that’s needed: 
 <a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/f/2f0b50db6e29bb2a41ea1ed96267ba00788a6bdd.png" data-download-href="/uploads/short-url/6IaGtVkJPxkKg07uX2MFVUQTE85.png?dl=1" title="image" rel="noopener nofollow ugc">[image]</a>
  </blockquote>
</aside>
</p>

**Comment by pattacini** [comment, marked as answer]
<p>Here’s the final official doc: <a href="https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#converting-issues-based-on-labels" rel="noopener nofollow ugc">https://docs.github.com/en/free-pro-team@latest/discussions/managing-discussions-for-your-community/managing-discussions-in-your-repository#converting-issues-based-on-labels</a>.</p>

**Comment by vrindanayak** [comment]
<p>Hello,</p>
<p>Thanks for the posts / links related to Github Discussions. We have enabled Discussions tab for 2 of our repositories. The problem is that converting an Issue to Discussion is enabled only in one of them (FYI : We didn’t explicitly configure any settings to enable this conversion from Issue to Discussion). We have not found any way yet, how this conversion from Issue to Discussion (<em>Convert to Discussion</em> link on bottom right of an Issue page) could be enabled also for the second repository. Are there any settings we need to look at?</p>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/dcm4che/dcm4che" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/1788145?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/dcm4che/dcm4che" target="_blank" rel="noopener nofollow ugc">dcm4che/dcm4che</a></h3>


  <p><span class="label1">DICOM Implementation in JAVA. Contribute to dcm4che/dcm4che development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/dcm4che/dcm4chee-arc-light" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:400/400;"><img src="https://avatars.githubusercontent.com/u/1788145?s=400&amp;amp;v=4" class="thumbnail" width="400" height="400"></div>

<h3><a href="https://github.com/dcm4che/dcm4chee-arc-light" target="_blank" rel="noopener nofollow ugc">dcm4che/dcm4chee-arc-light</a></h3>


  <p><span class="label1">DICOM Archive J2EE application. Contribute to dcm4che/dcm4chee-arc-light development by creating an account on GitHub.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/d/6de1964c66dc5f671110767b948f8dea71ab9127.jpeg" data-download-href="/uploads/short-url/fG3mt8wRikfY1oXcQ4aBHNNKhpl.jpeg?dl=1" title="dcm4che_dcm4chee-arc-light" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/15785031/181075477-bb5779b3-86f0-42db-859c-a927c47ff87f.jpeg" alt="dcm4che_dcm4chee-arc-light" data-base62-sha1="fG3mt8wRikfY1oXcQ4aBHNNKhpl" width="690" height="242" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6de1964c66dc5f671110767b948f8dea71ab9127_2_690x242.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6de1964c66dc5f671110767b948f8dea71ab9127_2_1035x363.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6de1964c66dc5f671110767b948f8dea71ab9127_2_1380x484.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/d/6de1964c66dc5f671110767b948f8dea71ab9127_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">dcm4che_dcm4chee-arc-light</span><span class="informations">3201×1125 473 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Any help is appreciated.</p>

**Comment by pattacini** [comment]
<p>Actually, I’ve been recently observing what you described: for some repos the conversion works, for others it doesn’t seem to be operational. I have honestly no clue about that.</p>
<p>The documentation doesn’t provide insights on this.</p>
